index.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>MajorStylin'</title>
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link type="text/css" rel="stylesheet" href="css/animate.css">
+    <link type="text/css" rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <nav>
+        <div class="wrapper">
+            <div id="mobilenav"><i class="fa fa-bars" aria-hidden="true"></i></div>
+            <a href="#" class="navlink">MAIN NAV</a>
+            <a href="#" class="navlink">LINK</a>
+            <a href="#" class="navlink">STYLE</a>
+            <a href="#" class="utilitylink">STYLE</a>
+            <a href="#" class="utilitylink">LINK</a>
+            <a href="#" class="utilitylink">UTILITY</a>
+        </div>
+    </nav>
+    <div class="spacer"> </div>
```

Comment:
Don't use things like "spacer". You should space things out with padding or margin.